trigger_class.py

```diff
@@ -0,0 +1,65 @@
+def contains(message, list_of_phrases):
+	"""
+	Checks if a message (string) contains all 
+	"""
+	for phrase in list_of_phrases:
+		if not contains_phrase(phrase):
+			return False
+
+	return True
+	
+
+def contains_phrase(message, phrase):
+	"""
+	Helper for checking if a phrase is in a message
+	"""
+	if phrase in message:
+		return True
+	return False
+
+#There needs to be more helper functions as deemed necessary
+
+class condition_wrapper():
+	def __init__(self, condition, output):
+		"""
+		Sets the condition of the condition_wrapper
+		"""
+		self.condition = condition
+		self.output = output
+	def check(self, message):
+		"""
+		Checks the message against the condition and returns None if false, output if true
+		"""
+		if self.condition(message):
+			return self.output
+		return None
+
+def list_check(message, conditions):
+	"""
+	Takes a list of condition wrappers
+	returns the output of the first wrapper that is true
+	"""
+	result = None
+	for condition in conditions:
+		result = condition.check(message)
+		if result is not None:
+			return result
+	return result
+
+if __name__ == "__main__":

```

Comment:
Probably make a more robust test case that actually checks for phrases instead of two conditions that just return straight variables